docs/README.md

```diff
@@ -6,7 +6,11 @@ This directory contains scripts to generate API Docs for Scala, Java, and Python
 Follow [Conda Installation Guide](https://docs.conda.io/projects/continuumio-conda/en/latest/user-guide/install/macos.html) to install Anaconda.
```

Comment:
This link didn't work for me. I just used https://www.anaconda.com/download/

Reply:
updated the link